rust/parquet/src/file/metadata.rs

```diff
@@ -433,6 +433,21 @@ impl ColumnChunkMetaData {
         self.dictionary_page_offset
     }
 
+    /// Returns the offset and length in bytes of the column chunk within the file
```

Comment:
Looks good to me. @sunchao  do you have any hesitation about hoisting this part of the code into the public API? I reviewed the logic carefully and it looks like a straight forward port